dotnet/src/SemanticKernel/Memory/ISemanticTextMemory.cs

```diff
@@ -15,32 +15,36 @@ public interface ISemanticTextMemory
     /// Save some information into the semantic memory, keeping a copy of the source information.
     /// </summary>
     /// <param name="collection">Collection where to save the information</param>
-    /// <param name="id">Unique identifier</param>
+    /// <param name="id">Unique identifier for searching memory metadata</param>
     /// <param name="text">Information to save</param>
     /// <param name="description">Optional description</param>
+    /// <param name="dbKey">The unique database key.</param>
     /// <param name="cancel">Cancellation token</param>
-    public Task SaveInformationAsync(
+    public Task<string> SaveInformationAsync(
```

Comment:
Could we add <returns> tags for methods in this interface that now return a value?